doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,160 @@
+(container-gpu-passthrough-with-docker)=
+# Passing an NVIDIA dGPU / iGPU to a LXD container with a nested Docker workload
+
+If you have NVIDIA runtime libraries and configuration installed on your host and you want to pass these files to your container, you should add a device with the CDI notation.
+Consider the following scenario:
+
+Your host is an NVIDIA single board computer that has a Tegra SoC with an iGPU, and you have an the Tegra SDK installed on the host. You want to create a LXD container and run an application inside the container using the iGPU as a compute backend. We want to run this application inside a Docker container (or another OCI-compliant runtime).
+To achieve this, complete the following steps:
+
+1. Initialize a LXD container with the following config keys (needed for running a Docker container inside a LXD container):
+
+        lxc init ubuntu:24.04 t1 --config security.nesting=true
+
+
+1. Add the GPU device to your container:
+
+        # If you want to do an iGPU passthrough
+        lxc config device add t1 igpu0 gpu gputype=physical id=nvidia.com/igpu=0
+
+        # Or, if you want to do an dGPU passthrough
+        lxc config device add t1 gpu0 gpu gputype=physical id=nvidia.com/gpu=0
+
+Let's try to run a basic [MNIST](https://en.wikipedia.org/wiki/MNIST_database) inference job inside our LXD container.
+
+1. For that, create a `cloud-init` script that installs the Docker runtime, the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit), and a script to run a test [TensorRT](https://github.com/NVIDIA/TensorRT) workload:
+
+        #cloud-config
+        package_update: true
+        packages:
+          - docker.io
+        write_files:
+          - path: /etc/docker/daemon.json
+            permissions: "0644"
+            owner: root:root
+            content: |
+            {
+                "max-concurrent-downloads": 12,
```

Comment:
i asked this previously, but are these required, or can we slim down the steps/config to the minimal required?

Reply:
it would be good to add a comment to explain changes made from defaults and why, but if not needed then could be removed to not indicate that its part of the actual howto.

Reply:
fixed